### PR TITLE
Update docs and cleanup folders

### PR DESCRIPTION
**Pull request by hemajv**
Addresses https://github.com/redhat-et/optimal-stopping-point-tool/issues/22

### CODE REVIEW

**Review by aakankshaduggal — COMMENTED** [review]
/lgtm
/approve

File: README.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
...
+
+This tool is part of a wider project that we have been working on called [AI4CI](https://github.com/aicoe-aiops/ocp-ci-analysis). AIOps is a critical component of supporting any Open Hybrid Cloud infrastructure. As the systems we operate become larger and more complex, intelligent monitoring tools and response agents will become a necessity.
+
+The problem that we are trying to address is that there is a need for automated monitoring, altering, AI driven analysis for operations and testing systems. Open source testing data originating from real world production systems is a rarity for public datasets. Also, we noticed a lack of AI driven metrics for open source community health. This brings us to the opportunity of leveraging datasets that are made available by running open source software and applications in production like openshift-ci, kubernetes, fedora and develop a collection of open source data science tools and AI models to collect and analyze the CI/CD data. Thus, enabling AI for CI. 
+
 ## Data Collection
 
 We collect historical test data mainly the test duration values from running workflows on Github and store the data on s3 storage.
```
Comment (oindrillac) [inline README.md:13]: The approach for calculating OSP has been revised, can we change the sections below to the reflect the updated approach? Approximating distributions and finding goodness of fit is no longer our primary approach so maybe we can add a section on previous experimental methods and link the notebook from there?

**Review by oindrillac — CHANGES_REQUESTED** [review]

File: notebooks/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+### Notebooks
+
+This folder contains various reproducible and interactive Jupyter notebooks created for the project.
+The notebook organizational structure is described in detail below.
+
+* [`access_github_ci_data.ipynb`](access_github_ci_data.ipynb) - Notebook which collects historical test data like the test duration values from running workflows on Github using the GitHub API.
+
+* [`test_distributions.ipynb`](test_distributions.ipynb) - Notebook which explores the statistical distributions of the test duration values to identify the best type of distribution for passing and failing tests.
```
[the remark this answers is not in the document]
Comment (hemajv) [reply notebooks/README.md]: updated 👍

File: README.md (hunk 1 above)
Comment (hemajv) [reply README.md:13]: Thanks for pointing that out. Update it 👍

**Review by oindrillac — APPROVED** [review]
Looks good, thanks for the changes!

**Pull request merged**